pkgs/by-name/sl/sleek-grub-theme/package.nix

```diff
@@ -0,0 +1,30 @@
+{pkgs, lib, ...}:
+pkgs.stdenv.mkDerivation rec {
+
+  pname = "sleek-grub-theme";
+  version = "unstable-2022-06-04";
+
+  src = pkgs.fetchFromGitHub ({
+    owner = "sandesh236";
+    repo = "sleek--themes";
+    rev = "981326a8e35985dc23f1b066fdbe66ff09df2371";
+    sha256 = "sha256-yD4JuoFGTXE/aI76EtP4rEWCc5UdFGi7Ojys6Yp8Z58=";
+  });
+
+  banner = "Grub Bootloader";
+
+  installPhase = ''
+    mkdir -p $out/
+
+    cp -r 'Sleek theme-dark'/sleek/* $out/
+    sed -i "s/Grub Bootloader/${banner}/" $out/theme.txt
```

Comment:
Why?

Reply:
To make sure the "banner" text is overrideable, 

From the uppersteam's [README](https://github.com/sandesh236/sleek--themes#preview):

> If you want to have a feeling of belonging, You can have your name on grub bootloader.

Reply:
It's from the uppersteam's install script  [here](https://github.com/sandesh236/sleek--themes/blob/master/Sleek%20theme-dark/install.sh#L88)

Reply:
In that case please add this as a function argument.